bugwarrior/services/gmail.py

```diff
@@ -178,12 +179,28 @@ def get_labels(self):
 
     def get_threads(self):
         thread_service = self.gmail_api.users().threads()
+        threads = []
 
-        result = thread_service.list(
-            userId=self.config.login_name, q=self.config.query).execute()
-        return [
-            thread_service.get(userId='me', id=thread['id']).execute()
-            for thread in result.get('threads', [])]
+        pageToken = None
+
+        while len(threads) <= self.config.email_limit:
+            maxResults = min(100, max(self.config.email_limit,
+                                      self.config.email_limit - len(threads)))
```

Comment:
Shouldn't this be `min(100, self.config.email_limit - len(threads))`?

The second `max` seems wrong (I think you meant `min`) but also redundant (`email_limit` will always be >= `email_limit - len(threads)`).